src/dev_server.rs

```diff
@@ -12,15 +12,19 @@ use std::{
 #[non_exhaustive]
 #[derive(Debug, Parser)]
 pub struct DevServer {
```

Comment:
You also need to provide documentation on structs. With clap it will also show in the `--help`.

Reply:
In progress